scss/mixins/_utilities.scss

```
@@ -21,12 +21,12 @@
     $property-class: if($property-class, $property-class, nth($properties, 1));
 
     // Don't prefix if value key is null (eg. with shadow class)
-    $property-class-modifier: if(nth($value, 1), "-" + nth($value, 1), "");
+    $property-class-modifier: if($key, "-" + $key, "");
 
     .#{$property-class + $infix + $property-class-modifier} {
       @each $property in $properties {
         // stylelint-disable-next-line declaration-no-important
-        #{$property}: #{nth($value, 2)} !important;
+        #{$property}: #{$value} !important;
```

Value should not be wrapped in `#{}` here:
```suggestion
        #{$property}: $value !important;
```